docs/src/pages/guides/right-to-left/RtlOptOut.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';
+
+declare module '@material-ui/core/styles/withStyles' {
```

Comment:
Let's be extra explicit and add that this is augmented by `jss-rtl`. While the information is available on the same page it's missing from this particular file and one page distance might already be too much :smile: